flexget/plugins/input/medusa.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+from future.moves.urllib.parse import urlparse
+
+import logging
+import json
+
+from requests import RequestException
+
+from flexget import plugin
+from flexget.event import event
+from flexget.entry import Entry
+
+log = logging.getLogger('medusa')
+
+
+class Medusa(object):
+    schema = {
+        'type': 'object',
+        'properties': {
+            'base_url': {'type': 'string'},
```

Comment:
you can set expected format to `uri`:
```suggestion
            'base_url': {'type': 'string', 'format': 'uri'},
```